data/incipias/silent harvesters.txt

```diff
@@ -0,0 +1,69 @@
+outfit "Strange Ray"
+	category "Guns"
+	mass 25
+	"outfit space" -20
+	"gun ports" -1
+	weapon
+		sprite "projectile/strange ray/ray"
+			"frame rate" 2
+		"inaccuracy" 0
+		"velocity" 1
+		"lifetime" 20
+		"random lifetime" 0
```

Comment:
```suggestion
```
I don't think a value of zero here does anything, the random lifetime should either be removed or given a nonzero value